glom/core.py

```diff
@@ -1821,7 +1834,7 @@ def glom(target, spec, **kwargs):
         else:
             err = GlomError.wrap(e)
         if isinstance(err, GlomError):
-            err._finalize(scope[ERROR_SCOPE])
+            err._finalize(scope[LAST_CHILD_SCOPE])
```

Comment:
Is `LAST_CHILD_SCOPE` a misnomer? As in, it's not the scope of the last child in the spec, it's actually the most-recently-evaluated spec? Not sure this helps much but would `LATEST_CHILD_SCOPE` be more accurate? More runtimey?

Reply:
yes, I'm trying to say "most recent" child

Reply:
Would "latest", "newest", or "current" be clearer?

Reply:
I dunno, its been around for awhile

```
LAST_CHILD_SCOPE.__doc__ = """
Marker that can be used by parents to keep track of the last child
scope executed.  Useful for "lifting" results out of child scopes
for scopes that want to chain the scopes of their children together
similar to tuple.
"""
```

I guess this is a new use case but renaming seems relatively low priority

Reply:
I guess the difference is if you consider the child completed or still in progress